Formula/jless.rb

```diff
@@ -12,6 +12,9 @@ def install
   end
 
   test do
-    system "#{bin}/jless", "--help"
+    (testpath/"example.json").write('{"hello": "world"}')
+    res, process = Open3.capture2("#{bin}/jless example.json")
+    assert_match(/"hello": "world"/, res)
```

Comment:
```suggestion
    assert_match('"hello": "world"', res)
```
You don't need a regex here.  Alternatively:
```suggestion
    assert_equal("world", JSON.parse(res)["hello"])
```